apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/RetryConformanceTest.cs

```diff
@@ -114,17 +96,17 @@ private async Task RunTestCase(InstructionList instructionList, Method method, b
                 {
                     RunRetryTest(response, context, method.Group, specifyPreconditions);
 
-                    //storage.buckets.setIamPolicy has no preconditions implemented in .NET and hence will retry successfully
+                    // storage.buckets.setIamPolicy has no preconditions implemented in .NET and hence will retry successfully
                     if (!method.Name.Contains("buckets.setIamPolicy"))
                     {
                         Assert.False(response.Completed);
                     }
                 }
                 catch (Exception ex) // To catch expected exception when retry should not happen.
                 {
-                    if (ex != null && ex is GoogleApiException)
+                    if (ex != null && ex is GoogleApiException exception)
                     {
-                        var statusCode = ((GoogleApiException) ex).HttpStatusCode;
+                        var statusCode = exception.HttpStatusCode;
 
                         if ((instructionList.Instructions.Contains("return-503") && statusCode == System.Net.HttpStatusCode.ServiceUnavailable)
```

Comment:
I hadn't spotted these before - can we just use `HttpStatusCode` rather than `System.Net.HttpStatusCode`? If there's a collision but we only ever need the System.Net type, we can have `using HttpStatusCode = System.Net.HttpStatusCode;` in the using directives.

Reply:
I don't know why this conversation was resolved - it doesn't seem to have been addressed.